src/app/pages/FrontPage/index.jsx

```diff
@@ -111,7 +111,7 @@ const FrontPage = ({ pageData, mostReadEndpointOverride }) => {
   return (
     <>
       {/* dotcom and dotcomConfig need to be setup before the main dotcom javascript file is loaded */}
-      {hasAds && !isAmp && <CanonicalAdBootstrapJs />}
+      {adsEnabled && !isAmp && <CanonicalAdBootstrapJs />}
```

Comment:
Ah it's not entirely clear here but `{!isAmp && <CanonicalAdBootstrapJs />}` would be fine. This bootstrap file only sets a few global variables required `if` ads are run, but otherwise it won't result in Ads being shown by itself.

Because the GNL script begins executing before it is enabled, this needs to be setup first or there is an odd race condition where it might load before the configuration is set.